rosbag2_storage/src/rosbag2_storage/base_io_interface.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
```

Comment:
with the logic of splitting solely being handled in the writer API, this change shouldn't be needed.